src/components/Menu/Menu.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import './Menu.css';
+import MenuLinks from './MenuLinks';
+import { uniqueNoOfDishes, getDish } from './MenuHelper';
+import Dishes from '../Dishes';
+import Cart from '../Cart';
+class Menu extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dishes: {},
+      quantities: [],
+      noOfDishes: 0,
+      cartItems: {
+        items: [],
+        total: 0,
+        totalItems: 0,
```

Comment:
total and total items can be calculated on the fly in child's component when required, so removing these two wuouldd make out state more simpler